lib/ansible_tower_client/base_models/project_update.rb

```diff
@@ -1,4 +1,8 @@
 module AnsibleTowerClient
   class ProjectUpdate < BaseModel
+    def stdout(format = 'txt')
+      api.get("#{related['stdout']}?format=#{format}").body
+    end
+    alias result_stdout stdout
```

Comment:
Why add the `alias`?

Reply:
the miq tower gem is depending on this alias